src/functions.php

```diff
@@ -13,8 +13,10 @@
  *
  * If `$promiseOrValue` is a promise, it will be returned as is.
  *
- * @param mixed $promiseOrValue
- * @return PromiseInterface
```

Comment:
Looks like this actually returns `ExtendedPromiseInterface`.

Reply:
Technically yes, but that isn't the base contract we're using. This is also why in v3 `ExtendedPromiseInterface` has been merged into `PromiseInterface`.